app/_landing_pages/inso-cli.yaml

```diff
@@ -0,0 +1,119 @@
+metadata:
+  title: Inso CLI
+rows:
+  - header:
+      type: h1
+      text: "Inso CLI"
+
+  - header:
+      type: h2
+      text: What is Inso CLI?
+    columns:
+    - blocks:
+        - type: text
+          config: |
+            Inso CLI is a command line tool that allows you to use Insomnia features in your terminal and CI/CD environments for automation.
+            
+  - header:
+      type: h2
+      text: "Install Inso CLI"
+    columns:
+      - blocks:
+        - type: tabs
+          config:
+            - title: OSX
+              include_content: tools/inso-cli/install/osx
+            - title: Windows
+              include_content: tools/inso-cli/install/windows
+            - title: Linux
+              include_content: tools/inso-cli/install/linux
+
+  - header:
+    columns:
+      - header:
+          type: h2
+          text: "How does Inso CLI work?"
+        blocks:
+        - type: text
+          config: |
+            When you run a command, Inso CLI will look for Insomnia data in two places:
+            * In the .insomnia directory in the working directory. This directory is created automatically when [Git sync]() is set up.
+            * In the Insomnia app data directory if Insomnia is installed.
+
+            You can also use the `--workingDir` or `--src` options to specify a different data source. For more information, see the [Inso CLI reference]().
+
+  - header:
```

Comment:
Could you add an example with the rest of the commands from [this table](https://docs.insomnia.rest/inso-cli/cli-command-reference#command-overview)? Just so we have all the main commands on this page.